public/doc/openapi-3-v1.0.0.json

```diff
@@ -665,6 +667,48 @@
               }
             }
           },
+          "304": {
+            "$ref": "#/components/responses/NotModified"
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Portfolio Not Found."
+          }
+        },
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/TagItem"
+              }
+            }
+          },
+          "required": true
+        }
+      }
+    },
+    "/portfolios/{id}/untag": {
+      "post": {
+        "tags": [
+          "Portfolio"
+        ],
+        "summary": "Delete Tags from Portfolio",
+        "description": "Delete Tags from Portfolio",
```

Comment:
Should summary, description etc use the term deleted?  Same with `PortfolioItem`.

Reply:
Changed to `Remove` i.e. "Remove Tags from Portfolio"